src/target/at32f43x.c

```diff
@@ -289,7 +304,66 @@ static bool at32f43_detect(target_s *target, const uint16_t part_id)
 	return true;
 }
 
-/* Identify AT32F43x "High Performance" line devices (Cortex-M4) */
+/* Identify AT32F405 Mainstream devices */
+static bool at32f405_detect(target_s *target, const uint32_t series)
+{
+	/*
+	 * AT32F405/F402 always contain 1 bank with 128 sectors
+	 * Flash (C): 256 KiB, 2 KiB per sector, 0x7005_3000
+	 * Flash (B): 128 KiB, 1 KiB per sector, 0x7004_2000
+	 */
+	const uint16_t flash_size = target_mem32_read16(target, AT32F4x_FLASHSIZE);
+	const uint16_t sector_size = (series == AT32F405_SERIES_128KB) ? 1024U : 2048U;
```

Comment:
The parens in this ternary can be dropped, and same for the matching one below in the next detect function. If you really want to keep them, they'd go to just before the closing `;`, but they are superfluous.

Reply:
Dropped parens in two ternaries.